content/about/feature-stages/index.md

```diff
@@ -85,24 +84,22 @@ Below is our list of existing features and their current phases. This informatio
 
 | Feature           | Phase
 |-------------------|-------------------
-| [Kubernetes: Envoy Installation and Traffic Interception](/docs/setup/kubernetes/)        | Beta
-| [Kubernetes: Istio Control Plane Installation](/docs/setup/kubernetes/) | Beta
+| [Kubernetes: Envoy Installation and Traffic Interception](/docs/setup/kubernetes/)        | Stable
+| [Kubernetes: Istio Control Plane Installation](/docs/setup/kubernetes/) | Stable
 | [Kubernetes: Istio Control Plane Upgrade](/docs/setup/kubernetes/) | Beta
 | [Pilot Integration into Kubernetes Service Discovery](/docs/setup/kubernetes/)         | Stable
+| [Pilot Integration into Consul](/docs/setup/consul/quick-start/) | Alpha
+| [Pilot Integration into Cloud Foundry Service Discovery](/docs/setup/consul/quick-start/)    | Alpha
 | [Attribute Expression Language](/docs/reference/config/policy-and-telemetry/expression-language/)        | Stable
 | [Mixer Adapter Authoring Model](/blog/2017/adapter-model/)        | Stable
-| [VM: Envoy Installation, Traffic Interception and Service Registration](/docs/examples/integrating-vms/)    | Alpha
-| [VM: Istio Control Plane Installation and Upgrade (Galley, Mixer, Pilot, CA)](https://github.com/istio/istio/issues/2083)  | Alpha
+| [VM: Envoy Installation, Traffic Interception and Service Registration](/docs/examples/integrating-vms/)    | Beta
 | VM: Ansible Envoy Installation, Interception and Registration  | Alpha
-| [Pilot Integration into Consul](/docs/setup/consul/quick-start/) | Alpha
-| [Pilot Integration into Cloud Foundry Service Discovery](/docs/setup/consul/quick-start/)    | Alpha
 | [Basic Config Resource Validation](https://github.com/istio/istio/issues/1894) | Alpha
-| [Mixer Telemetry Collection (Tracing, Logging, Monitoring)](/help/faq/mixer/#mixer-self-monitoring) | Alpha
+| [Mixer Self Monitoring](/help/faq/mixer/#mixer-self-monitoring) | Beta
 | [Custom Mixer Build Model](https://github.com/istio/istio/wiki/Mixer-Compiled-In-Adapter-Dev-Guide) | Alpha
-| [Out of Process Mixer Adapters](https://github.com/istio/istio/wiki/Out-Of-Process-gRPC-Adapter-Dev-Guide) | Alpha
-| Enable API attributes using an IDL | Alpha
+| [Out of Process Mixer Adapters (GRPC Adapters)](https://github.com/istio/istio/wiki/Out-Of-Process-gRPC-Adapter-Dev-Guide) | Alpha
 | [Helm](/docs/setup/kubernetes/helm-install/) | Beta
-| [Multicluster Mesh](/docs/setup/kubernetes/multicluster-install/) | Alpha
+| [Multicluster Mesh](/docs/setup/kubernetes/multicluster-install/) | Beta
```

Comment:
Not sure this is beta. Is it?

Reply:
@costinm and I discussed this yesterday and felt that given the extensive testing that Cisco has done and that it has been available for use since 0.8 without any issues and decent documentation, it's relatively safe to promote it to beta.